mod/tagger.php

```diff
@@ -97,7 +97,7 @@ function tagger_content(&$a) {
 	if(! isset($bodyverb))
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊